docs: document only CLI flows (#2319)
codenvy/che-docs: modules/administration-guide/pages/configuring-routes.adoc

[id="configuring-routes_{context}"]
= Configuring OpenShift Route to work with Router Sharding

> This procedure describes how to configure labels, annotations, and domains for OpenShift Route to work with link:https://docs.openshift.com/container-platform/4.7/networking/ingress-operator.html#nw-ingress-sharding_configuring-ingress[Router Sharding].

Reviewer: ```suggestion
You can configure labels, annotations, and domains for OpenShift Route to work with link:https://docs.openshift.com/container-platform/4.7/networking/ingress-operator.html#nw-ingress-sharding_configuring-ingress[Router Sharding].
```
IBM SG: 
Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes". Instead, focus on the purpose or value of the content, and include words that users are likely to search for. For example, in a short description in an online help topic, provide users with enough information to determine whether the topic meets their needs. You can use self-referential text in learning modules and tutorials.

> <2> DNS name serviced by the target ingress controller
> <3> unstructured key value map stored with a resource

Reviewer: ```suggestion
<1> A comma-separated list of labels that the target ingress controller uses to filter the set of Routes to service.
<2> The DNS name serviced by the target ingress controller.
<3> An unstructured key value map stored with a resource.```

> .Additional resources

Reviewer: I assume that the openshift-docs requirement to place an ID above the additional resources does not apply here.

Author: Yes

> * xref:understanding-the-checluster-custom-resource.adoc[]

Reviewer: I assume the Che documentation allows xrefs in "modules" and that these xrefs do not have the same syntax as in openshift-docs.

Author: Yes
